docs/website/docs/getting-started/tflite.md

```diff
@@ -0,0 +1,54 @@
+# TFLite Integration
+
+IREE supports coming and running TensorFlow-Lite programs stored as [TFLite
+flatbuffers](https://www.tensorflow.org/lite/guide). These files can be
+imported into an IREE compatible format then compiled to a series of backends.
+
+## Prerequisites
+
+Install TensorFlow-Lite specific dependencies using pip:
+
+```shell
+python -m pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tflite
+```
+
+- [Command Line](./tflite-cmd.md)
+- [Python API](./tflite-python.md)
+
+## Troubleshooting
+
+Failures during the import step usually indicate a failure to lower from 
+TensorFlow Lite's operations to TOSA, the intermediate representation used by
+IREE. Many TensorFlow Lite operations are not fully supported, particularly
+those than use dynamic shapes. File and issue to IREE's TFLite model support
+[project](https://github.com/google/iree/projects/42). 
```

Comment:
we could be a bit more optimistic with phrasing than "Many TensorFlow Lite operations are not fully supported"
maybe "Some" instead of "Many"? :P

Reply:
Done.